rocks/RocksTypeDB.java

```diff
@@ -1,114 +0,0 @@
-/*
```

Comment:
This class has been deleted, and have the main functionalities moved to `CoreDatabaseManager`.